arch/lkl/kernel/threads.c

```diff
@@ -5,7 +5,7 @@
 #include <asm/cpu.h>
 #include <asm/sched.h>
 
-static volatile int threads_counter;
```

Comment:
@thehajime The patch doesn't work for me right now. 
After changing to `lkl__sync_` functions, the `__sync_synchronize` used in `lib/virtio.c` becomes undefined once again.  Then, I tried to change it to `lkl__sync_synchronize` and compile, what I got is that there're two `lkl__sync_synchronize` symbols in `liblkl-hijack.so`: 
```
00061454 l     F .text	00000044              lkl__sync_synchronize
00000000         *UND*	00000000              lkl__sync_synchronize
```
The undefined symbol comes from `lib/lkl-in.o`, the defined one comes from `lib/lkl.o`. When running `AR` to generate `liblkl.a` based on `lkl-in.o` and `lkl.o`,  the two symbols are joint together. 

What should I do to get rid of it? I am not professional on GCC toolchain....